ui/apps/platform/cypress/integration/dashboard/dashboardSecurityMetrics.test.js

```diff
@@ -0,0 +1,97 @@
+import * as api from '../../constants/apiEndpoints';
+import withAuth from '../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../helpers/features';
+import { visitMainDashboardPF } from '../../helpers/main';
+
+import { pfSelectors as selectors } from '../../constants/DashboardPage';
+
+function makeFixtureCounts(crit, high, med, low) {
+    return [
+        { severity: 'CRITICAL_SEVERITY', count: `${crit}` },
+        { severity: 'HIGH_SEVERITY', count: `${high}` },
+        { severity: 'MEDIUM_SEVERITY', count: `${med}` },
+        { severity: 'LOW_SEVERITY', count: `${low}` },
+    ];
+}
+
+const policyViolationsByCategory = {
+    groups: [
+        { counts: makeFixtureCounts(5, 20, 30, 10), group: 'Anomalous Activity' },
+        { counts: makeFixtureCounts(5, 2, 30, 5), group: 'Docker CIS' },
+        { counts: makeFixtureCounts(10, 20, 5, 5), group: 'Network Tools' },
+        { counts: makeFixtureCounts(15, 2, 10, 5), group: 'Security Best Practices' },
+        { counts: makeFixtureCounts(20, 10, 2, 10), group: 'Privileges' },
+        { counts: makeFixtureCounts(15, 8, 10, 5), group: 'Vulnerability Management' },
+    ],
+};
+
+describe('Dashboard security metrics phase one action widgets', () => {
+    withAuth();
+
+    before(function beforeHook() {
+        if (!hasFeatureFlag('ROX_SECURITY_METRICS_PHASE_ONE')) {
+            this.skip();
+        }
+    });
+
+    it('should visit patternfly dashboard', () => {
+        visitMainDashboardPF();
+
+        cy.get(selectors.pageHeader);
+    });
+
+    it('should sort a policy violations by category widget by severity and volume of violations', () => {
+        visitMainDashboardPF();
+
+        cy.intercept('GET', api.alerts.countsByCategory, {
```

Comment:
Aha, your thoughts are welcome for a clear pattern to distinguish:
* Tests of other containers that call `visitMainDashboard` need the updated function to wait on relevant requests before interactive action.
* Tests of dashboard need to mock relevant requests. If widgets are responsible for own data fetching, other requests do not matter for DOM assertions. However, they might matter to prevent next test from starting before requests have completed for current test. For example, compliance tests did not fail, but cypress side panel suggested several false starts.